packages/block-editor/src/components/block-mobile-toolbar/index.js

```diff
@@ -8,18 +8,15 @@ import { useViewportMatch } from '@wordpress/compose';
  */
 import BlockMover from '../block-mover';
 
-function BlockMobileToolbar( { clientId, moverDirection } ) {
+function BlockMobileToolbar( { clientId } ) {
```

Comment:
I think this file can actually be deleted. I couldn't see any usage other than the native mobile version of it.

Reply:
Wouldn't deleting it break the native mobile app, then?

Reply:
Yeah, that's what I don't know. I don't think so as there's a separate index.native.js.

Reply:
Aha, gotcha. Yeah you're right that it probably can be removed.